h/buildext.py

```diff
@@ -218,45 +187,84 @@ def build_chrome(args):
     copytree('h/browser/chrome/help', 'build/chrome/help')
     copytree('h/browser/chrome/images', 'build/chrome/images')
     copytree('h/static/images', 'build/chrome/public/images')
+    copytree('h/static/styles/vendor/fonts', 'build/chrome/public/fonts')
 
-    os.makedirs('build/chrome/lib')
+    subprocess_args = ['node_modules/.bin/gulp', 'build']
+    subprocess.call(subprocess_args)
 
-    subprocess_args = ['node_modules/.bin/browserify',
-                       'h/browser/chrome/lib/extension.js', '--outfile',
-                       'build/chrome/lib/extension-bundle.js']
+    os.makedirs('build/chrome/lib')
+    shutil.copyfile('build/scripts/extension.bundle.js',
+                    'build/chrome/lib/extension.bundle.js')
     if args.debug:
-        subprocess_args.append('--debug')
-    subprocess.call(subprocess_args)
+        shutil.copyfile('build/scripts/extension.bundle.js.map',
+                        'build/chrome/lib/extension.bundle.js.map')
+
+    webassets_env = AssetEnvironment('/public')
+    webassets_env.register('app_js', [
```

Comment:
This duplicates a bunch of configuration from `h/assets.yaml`, which is now essentially a dictionary mapping names of bundle collections (eg. `app_js` to bundle names). I think it would probably make sense to load in the existing `assets.yaml` file and use that. Might even make sense to convert `h/assets.yaml` to a simple JSON file and implement a trivial webassets loader for it.